db/migrate/20220923200304_change_duration_to_int.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class ChangeDurationToInt < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :movies, :duration
```

Comment:
This migration is not reversible. It would be if you add the type of column like it was before. We should create migrations that are reversible :) You can try if it is with `rails db:rollback` 
```suggestion
    remove_column :movies, :duration, :interval
```